packages/twenty-front/src/modules/auth/components/VerifyEmailEffect.tsx

```diff
@@ -0,0 +1,65 @@
+import { useAuth } from '@/auth/hooks/useAuth';
+import { AppPath } from '@/types/AppPath';
+import { SnackBarVariant } from '@/ui/feedback/snack-bar-manager/components/SnackBar';
+import { useSnackBar } from '@/ui/feedback/snack-bar-manager/hooks/useSnackBar';
+
+import { useReadCaptchaToken } from '@/captcha/hooks/useReadCaptchaToken';
+import { useEffect, useState } from 'react';
+import { useNavigate, useSearchParams } from 'react-router-dom';
+import { EmailVerificationSent } from '../sign-in-up/components/EmailVerificationSent';
+
+export const VerifyEmailEffect = () => {
+  const { getLoginTokenFromEmailVerificationToken } = useAuth();
+
+  const { enqueueSnackBar } = useSnackBar();
+
+  const [searchParams] = useSearchParams();
+  const [isError, setIsError] = useState(false);
+  const email = searchParams.get('email');
+  const emailVerificationToken = searchParams.get('emailVerificationToken');
+
+  const navigate = useNavigate();
+  const { readCaptchaToken } = useReadCaptchaToken();
+
+  useEffect(() => {
```

Comment:
![image](https://github.com/user-attachments/assets/22ba72a0-3836-4e7a-8a18-8b8d8b5a580c)

@FelixMalfait I've tried including adding a guard in useEffect, minimizing dependencies, and checking for duplicate renders or snackbar triggers in other components. 

Despite this, the issue of duplicate snackbars persists and seems to only occur in development mode when Strict Mode is active. Is it fine to leave this behavior as is for now?

Reply:
Hey, no it's not acceptable - we need to find an other solution

Reply:
It will not happen in prod env. It's expected in dev env with strict mode enable.

Check the note here: https://react.dev/reference/react/StrictMode

Reply:
@AMoreaux yes agree but it's not an acceptable behavior in dev env. It means there's a code smell / we shouldn't rely on useEffect here